README.md

```diff
@@ -8,21 +8,22 @@ Keywords: Reinforcement learning, AWS, RoboMaker
 
 ![object-tracker-world.jpg](docs/images/object-tracker-world.jpg)
 
+
 ## Requirements
 
-- ROS Kinetic / Melodic (optional) - To run the simulation locally. Other distributions of ROS may work, however they have not been tested
-- Gazebo (optional) - To run the simulation locally
+- ROS Dashing (optional) - To run the simulation locally. Other distributions of ROS may work, however they have not been tested
```

Comment:
Why is dashing optional?

Reply:
Not sure why Kinetic/Melodic was optional for the ROS1 version as well.

It is probably because if simulating remotely on RoboMaker, `colcon bundle` would have already packaged ROS into it, even though `colcon build` would still have needed ROS.

I agree that this is quite confusing.

Reply:
I think it'd be best if we removed it from both version, but for the scope of this PR I'm fine with leaving it for consistency

Reply:
I just removed (optional) from this ROS2 version of the README.

The ROS1 version of the README definitely also needs a cleanup, with more issues than just (optional).